mediacloud/mediawords/util/web/test_user_agent.py

```diff
@@ -1032,6 +1037,14 @@ def test_get_http_auth(self):
 
         hs.stop()
 
+    def test_get_request_invalid_url(self):
+        """request() with invalid URL."""
+        with pytest.raises(McUserAgentRequestException):
+            Request(method='GET',
+
```

Comment:
inadvertent space? also, the comment can be passed into `pytest.raises`: 

```
with pytest.raises(McUserAgentRequestException, message='Expected "Invalid IDNA error"'):
...
```

Reply:
Thanks again, updated to just a comment (as `message` is not much used elsewhere).